tests/unit/test_dataset_by_external_id.py

```diff
@@ -0,0 +1,53 @@
+import json
+
+import pytest
+import responses
+
+from tamr_unify_client import Client
+from tamr_unify_client.auth import UsernamePasswordAuth
+
+_dataset_json = """[{
```

Comment:
If you want to avoid manually `dumps` and `loads` JSON, you could instead represent this as Python (i.e. `list`s and `dict`s) and use `responses.add(..., json=_dataset_json)` and then compare directly to `.data` (i.e. `actual_dataset.data == _dataset_json`).

Reply:
👍